unittests/test_rest_framework.py

```diff
@@ -1586,7 +1586,7 @@ def __init__(self, *args, **kwargs):
         }
         self.update_fields = {"first_name": "test changed", "configuration_permissions": [219, 220]}
         self.test_type = TestType.CONFIGURATION_PERMISSIONS
-        self.deleted_objects = 19
+        self.deleted_objects = 25
```

Comment:
Is this increase due to more Alert objects existing in the db due to better notification coverage?

Reply:
Yes, exactly.